plugins/woocommerce/tests/legacy/unit-tests/session/class-wc-tests-session-handler.php

```
@@ -252,6 +252,70 @@ public function test_destroy_session_user_mismatch() {
+
+		$handler = $this
+			->getMockBuilder( WC_Session_Handler::class )
+			->setMethods( array( 'get_session_cookie' ) )
+			->getMock();
+
+		$handler
+			->method( 'get_session_cookie' )
+			->willReturn( array( $guest_session_id, $session_expiration, $session_expiring, 'cookie_hash' ) );
+
+		$session_cookie_value = null;
+		$capture_cookie       = function ( $enabled, $name, $value ) use ( &$session_cookie_value ) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$enabled'. (undefined)

(UnusedFormalParameter)